packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -2,22 +2,28 @@
  * WordPress dependencies
  */
 import { Button } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
-import { withDispatch } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import { withDispatch, useSelect } from '@wordpress/data';
 
 const SettingsHeader = ( {
 	openDocumentSettings,
 	openBlockSettings,
 	sidebarName,
 } ) => {
-	const blockLabel = __( 'Block' );
+	const documentLabel = useSelect( ( select ) => {
+		const settings = select( 'core/block-editor' ).getSettings();
```

Comment:
As said above, if you check the component  I link previously, you'll see that we already have the info in the post type, we don't  a new setting for that.